### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by h4570**
Based on #155 by @Wolf3s

### CODE REVIEW

File: engine/inc/loaders/texture/builder/texture_builder_data.hpp [hunk 1]
```diff
@@ -24,14 +24,14 @@ class TextureBuilderData {
   std::string name;
 
   int width, height;
-  u8* data;
+  unsigned char* data;
   TextureBpp bpp;
```
Comment (Wolf3s) [inline engine/inc/loaders/texture/builder/texture_builder_data.hpp:28]: So it's here...

Comment (Wolf3s) [reply engine/inc/loaders/texture/builder/texture_builder_data.hpp:28]: Well it was really my mistake

Comment (h4570) [reply engine/inc/loaders/texture/builder/texture_builder_data.hpp:28]: `unsigned char` makes code more portable.
It was my fault to put `u8`, `u16` and other custom types in all tyra source code...

Comment (Wolf3s) [reply engine/inc/loaders/texture/builder/texture_builder_data.hpp:28]: I know that, I was thinking changing that for u8 due to it wasn't ready the multiplataform support from that time but seeing back then i should kept lol.

**Pull request merged**